frontend/src/routes/Search/Details/DetailsOverviewPage.tsx

```diff
@@ -321,9 +323,12 @@ export default function DetailsOverviewPage() {
   }, [cluster, resource, navigate])
 
   const { acmExtensions } = useContext(PluginContext)
-  let VirtualMachinesOverviewTab
+  let VirtualMachinesOverviewTab: React.ComponentType<any> | undefined
+  let KubevirtPluginContext: Context<KubevirtPluginData> | undefined
+  // TODO: these need to be found via unique ID
   if (acmExtensions?.searchDetails && acmExtensions.searchDetails.length) {
     VirtualMachinesOverviewTab = acmExtensions.searchDetails[0].properties.component
+    KubevirtPluginContext = acmExtensions.searchDetails[1].properties.context as Context<KubevirtPluginData>
```

Comment:
Put this code in the wrapper, we don't want to have to add it each time we access the extension data.

Reply:
This change is complete.